src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -433,59 +427,6 @@
     (throw (IllegalArgumentException.
              "Error: More than one default server specified in configuration"))))
 
-(defn selectors-count
```

Comment:
Production code wouldn't use these anymore however the count functions are still being used in the tests in the jetty9-default-config-test namespace to help validate that Jetty is using the algorithm that we think it is for calculating acceptor and selector threads.